### PR TITLE
Add Codenvy to the list of xTerm supporters

### PR DESCRIPTION
**Pull request by TylerJewell**
This is the follow-on PR per @Tyriar and @parisk.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +121,11 @@ Visit https://lair.io/sourcelair/xterm and follow the instructions. All developm
 
 #### [Eclipse Che](http://www.eclipse.org/che)
 
-You can start Eclipse Che with `docker run eclipse/che start`
+You can start Eclipse Che with `docker run eclipse/che start`.
```
Comment (Tyriar) [inline README.md]: @parisk should we remove the "Known tools that use xterm.js internally" list as it pretty much duplicates the real world uses?

**Comment by TylerJewell** [comment]
It did strike me as odd that there were two sections - I couldn't really differentiate the meaning of the two.

**Comment by Tyriar** [comment]
I initially thought it was:

- Here are products that use xterm.js
- Here are tools you can use to develop it

The second section isn't really necessary though imo.

**Comment by TylerJewell** [comment]
Just to keep things a bit more organized, with Eclipse Che, we have had to move anything related to making a contribution into either a CONTRIBUTION.md file (the highlights) and if you are an active developer of the project, we move all of the development workflow into a series of documents contained within the project's wiki.  It's harder to maintain, but it keeps the initial README a bit simpler to get your arms around, and then have it more focused on the end user than a project contributor.

**Comment by parisk** [comment]
Yep, now that it started getting bigger, I think it pretty much looks like a duplicate.

I will move forward with merging it and I will open up a new PR to rephrase things.

Thanks @TylerJewell for your contribution!

**Pull request merged**